sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+## 1.3.2 (TBA)
+
+### Features Added
+
+- Added inline image support
+- Updated @azure/communication-signaling to 1.0.0-beta.17 with inline image support.
```

Comment:
I'm not sure I follow what this entry means when we are in @azure/communication-chat

> Updated @azure/communication-signaling to 1.0.0-beta.17 with inline image support

Also, can you please change (TBA) to (Unreleased)?

Reply:
Good question @joheredi, Chat SDK is a wrapper that does not have internal logic. The actual logic (where it does encoding and decoding Trouter payload) is handled in its dependency @azure/communication-signaling. So to support inline image, its dependency would also needs to be updated. 

the reason why I call it out in our changelog is because I noticed we always disclose this: 
<img width="584" alt="image" src="https://user-images.githubusercontent.com/109105353/227615018-75d328dc-4390-47fa-8f15-45568222fa89.png">